paper-server/patches/sources/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch

```diff
@@ -376,7 +376,7 @@
  
              this.player
 -                .absSnapTo(
-+                 .snapTo( // Paper - Fix Entity Teleportation and cancel velocity if teleported
++                 .absSnapTo(
```

Comment:
This bit of diff can be fully removed, as the only leftover changed part seems to be a single space